src/pages/Home.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import 'bootstrap/dist/js/bootstrap';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import { Carousel } from 'react-bootstrap';
+import image from '../assets/images/pink1.jpg';
+import rates2 from '../assets/images/pink2.jpg';
+import rates3 from '../assets/images/pink3.jpg';
+
+const Home = () => (
+  <div style={{ overflow: 'none' }}>
+    <Carousel fade>
+      <Carousel.Item>
+        <img
+          src={image}
+          className="d-block w-100"
+          alt="stock"
+          style={{ height: '100vh' }}
+        />
+        <Carousel.Caption>
+          <h3>Check accurate coin data 💯</h3>
+          <p>We provide accurate coin data all over the world.</p>
+        </Carousel.Caption>
+      </Carousel.Item>
+      <Carousel.Item>
+        <img
+          src={rates2}
+          alt="stock2"
+          className="d-block w-100"
+          style={{ height: '100vh' }}
+        />
+        <Carousel.Caption>
+          <h3>Check data of all coins 😇</h3>
```

Comment:
- [ ] Nice try to follow the request by the previous review however you should put two cards in the  row as in the design provided 
- You should remove the space between cards as in the design provided 
<img width="883" alt="Screen Shot 2022-07-26 at 13 24 43" src="https://user-images.githubusercontent.com/61790587/181006522-a9022f90-e7e1-4028-afc3-62fe3235e9ad.png">

- Yours
<img width="879" alt="Screen Shot 2022-07-26 at 13 24 30" src="https://user-images.githubusercontent.com/61790587/181006262-244a58a1-796a-4036-83f7-15e71cde3f7c.png">
- Design 
<img width="242" alt="Screen Shot 2022-07-26 at 13 31 36" src="https://user-images.githubusercontent.com/61790587/181006395-f4af4701-e49c-4931-80d9-0ec7d6c75aa9.png">